src/arrayUtils.js

```diff
@@ -1,5 +1,9 @@
 // All functions here should be maybeExported from jsifier.js
 
+if (typeof maybeAssert !== 'function') {
+  function maybeAssert() {}
```

Comment:
I think this actually might not work in all browsers, because it's not well defined in the spec. Maybe better is to do
````
var maybeAssert;
if (typeof maybeAssert !== 'function') {
  maybeAssert = ..
````
The `var` shouldn't interfere with the proper declaration elsewhere if there is one.
````